5calls/app/src/main/java/org/a5calls/android/a5calls/controller/StatsActivity.java

```diff
@@ -0,0 +1,124 @@
+package org.a5calls.android.a5calls.controller;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.util.Pair;
+import android.support.v7.app.AppCompatActivity;
+import android.text.TextUtils;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.a5calls.android.a5calls.AppSingleton;
+import org.a5calls.android.a5calls.R;
+import org.a5calls.android.a5calls.model.DatabaseHelper;
+import org.a5calls.android.a5calls.model.Issue;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Tell the user how great they are!
+ */
+// TODO: Add sharing
+public class StatsActivity extends AppCompatActivity {
+    private static final String TAG = "StatsActivity";
+
+    @BindView(R.id.no_calls_message) TextView noCallsMessage;
+    @BindView(R.id.stats_holder) LinearLayout statsHolder;
+    @BindView(R.id.your_call_count) TextView callCountHeader;
+    @BindView(R.id.stats_contacted) TextView statsContacted;
+    @BindView(R.id.stats_vm) TextView statsVoicemail;
+    @BindView(R.id.stats_unavailable) TextView statsUnavailable;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_stats);
+        ButterKnife.bind(this);
+
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        DatabaseHelper db = AppSingleton.getInstance(this).getDatabaseHelper();
+        int count = db.getCallsCount();
+        if (count == 0) {
+            // Show a "no impact yet!" message.
+            noCallsMessage.setVisibility(View.VISIBLE);
+            statsHolder.setVisibility(View.GONE);
+            return;
+        }
+        noCallsMessage.setVisibility(View.GONE);
+        statsHolder.setVisibility(View.VISIBLE);
+        callCountHeader.setText(count == 1 ?
+                getResources().getString(R.string.your_call_count_one) :
+                String.format(getResources().getString(R.string.your_call_count), count));
+
+        int contactCount = db.getCallsCountForType(IssueActivity.CONTACTED);
+        int vmCount = db.getCallsCountForType(IssueActivity.VOICEMAIL);
+        int unavailableCount = db.getCallsCountForType(IssueActivity.UNAVAILABLE);
+        statsContacted.setText(contactCount == 1 ?
```

Comment:
I generally like to split the UI update logic into helpers to keep onCreateView from getting too large.  I'll usually have one or more of the following: `init()`, `updateUI(models,...)`, `refreshData()`.  In this case, since the page is static and the data is coming from the db, you would probably only have `updateUI(model)`, or you could even make it `init(model)`.